lib/list.js

```diff
@@ -360,12 +360,13 @@ function nodeReleaseInfoToVersion(remoteName, remoteUri, release) {
 				ext = '.7z';
 			}
 		} else {
-            // Official builds before 0.12.10 or 0.10.42 did not publish .tar.xz packages.
+            // Official builds before 0.12.10, 0.10.42 or 0.8.6 did not publish .tar.xz packages.
 			if (process.env['NVS_USE_XZ'] === '1' && !(
 					/^0\.12\.[0-9]$/.test(semanticVersion) ||
 					/^0\.11\./.test(semanticVersion) ||
 					/^0\.10\.4[0-1]$/.test(semanticVersion) ||
-					/^0\.10\.[1-3]?[0-9]$/.test(semanticVersion))) {
+					/^0\.10\.[1-3]?[0-9]$/.test(semanticVersion) ||
+					/^0\.8\.([6-9]|[1-2][0-9])$/.test(semanticVersion))) {
 				ext = '.tar.xz';
```

Comment:
This condition is to **not** match these versions. If it is `0.8.26` - `0.8.28`, this condition will fail and we will use `.gz` as intended.